parsl/monitoring/visualization/plots/default/workflow_plots.py

```diff
@@ -203,7 +203,13 @@ def workflow_dag_plot(df_tasks, group_by_apps=True):
         groups_list = {app: (i, None) for i, app in enumerate(
             df_tasks['task_func_name'].unique())}
     else:
-        groups_list = {'Pending': (0, 'gray'), "Running": (1, 'blue'), 'Completed': (2, 'green'), 'Unknown': (3, 'red')}
+        groups_list = {"unsched": (0, 'gray'),
+                       "pending": (1, 'gray'),
+                       "running": (2, 'blue'),
+                       "dep_fail": (3, 'red'),
+                       "failed": (4, 'red'),
+                       "done": (5, 'green')
+                      }
```

Comment:
I see the gantt chart already has a color scheme for different states. It seems better to port that scheme here? Or we do not care to show different colors for different states?

Reply:
you are right - this should use the same colour scheme for consistency

Reply:
@ZhuozhaoLi I've changed to use a consistent colour scheme with the gantt chart, and added in a missing state.